web-ui/src/main/resources/catalog/views/default/templates/home.html

```diff
@@ -44,23 +44,20 @@ <h1 data-translate="">getstarted</h1>
 
             An icon could be provided as a CSS class name or
             a pattern matching a URL resolving to an image.
-
-            Properties
-            <div data-flfilter="categories"
-               data-title=""
-               data-icon-path="../../images/category/{{['@name']}}.png">
             -->
           <div class="row">
-            <div data-ng-repeat="(key, facet) in searchInfo.facet['topicCats']"
-              class="col-xs-6 col-sm-4 col-lg-3">
-              <a class="thumbnail gn-icon-bg" data-ng-href="#/search?topicCat={{facet['@name']}}">
-                <i class="fa fa-3x fa-table gn-icon gn-icon-{{facet['@name']}}"></i><br/>
-                <span class="gn-icon-label">{{facet['@label']}}</span>
+            <div data-ng-repeat="(key, facet) in searchInfo.facet['orgNames'] | limitTo:12"
```

Comment:
Aren't the facets controlled in the config-summary.xml file?

Jesse

On Thu, Feb 26, 2015 at 4:45 PM, paul van genuchten <
notifications@github.com> wrote:

> In web-ui/src/main/resources/catalog/views/default/templates/home.html
> <https://github.com/geonetwork/core-geonetwork/pull/824#discussion_r25436525>
> :
>
> >              -->
> >            <div class="row">
> > -            <div data-ng-repeat="(key, facet) in searchInfo.facet['topicCats']"
> > -              class="col-xs-6 col-sm-4 col-lg-3">
> > -              <a class="thumbnail gn-icon-bg" data-ng-href="#/search?topicCat={{facet['@name']}}">
> > -                <i class="fa fa-3x fa-table gn-icon gn-icon-{{facet['@name']}}"></i><br/>
> > -                <span class="gn-icon-label">{{facet['@label']}}</span>
> > +            <div data-ng-repeat="(key, facet) in searchInfo.facet['orgNames'] | limitTo:12"
>
> Sure, I would like the topicCats, however the default q-search doesn't
> return them (at least for me) so probably better to fix that, if you have
> an idea...
>
> I thought about that, moving the type of facets to display on homepage to
> a config parameter, should be possible, because it's a quite generic module
>
> NGR used to have org-logo's which was quite nice
>
> Yes, I like the INSPIRE theme's here I'll ask Max to look for font-icons
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geonetwork/core-geonetwork/pull/824/files#r25436525>.
>